master/static/migrations/20240112140933_add-storage-config.tx.up.sql

```diff
@@ -0,0 +1,190 @@
+CREATE TABLE storage_backend_shared_fs (
+  id integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+  host_path        TEXT NOT NULL,
+  container_path   TEXT,
+  checkpoint_path  TEXT,
+  tensorboard_path TEXT,
+  storage_path     TEXT,
+  propagation      TEXT NOT NULL
+  CONSTRAINT shared_fs_reserved_value CHECK (
+    container_path != 'DeterminedReservedNullUniqueValue' AND
+    checkpoint_path != 'DeterminedReservedNullUniqueValue' AND
+    tensorboard_path != 'DeterminedReservedNullUniqueValue' AND
+    storage_path != 'DeterminedReservedNullUniqueValue'
+  )
+);
+
+-- Hack for lack of pre-postgres 15 NULLS NOT DISTINCT.
+-- https://stackoverflow.com/questions/8289100/create-unique-constraint-with-null-columns
```

Comment:
thank you for this comment

Reply:
we should bring up the discussion about increasing the minimum version of postgres

Reply:
I can add it to the old ticket https://hpe-aiatscale.atlassian.net/browse/DET-8931